packages/primitives/src/Signature.ts

```
@@ -0,0 +1,222 @@
+  v: number
+): Effect.Effect<Signature, Error> =>
+  Effect.gen(function*(_) {
+    const yParity = v % 2 === 0 ? 0 : 1;
+    
+    return {
+      r,
+      s,
+      v,
+      yParity: yParity as (0 | 1)
+    };
+  });
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)